.vib/joomla/goss/goss.yaml

```
@@ -0,0 +1,26 @@
+file:
+  /opt/bitnami/joomla:
+    filetype: symlink
+    linked-to: /bitnami/joomla
+    exists: true
+  /bitnami/joomla/logs:
+    exists: true
+    filetype: directory
+    mode: "0775"
+  /bitnami/joomla/configuration.php:
+    exists: true
+    filetype: file
+    mode: "0664"
+    contains:
+      - {{ printf "$db = '%s'" .Env.JOOMLA_DATABASE_NAME }}
```

This is fine, but I personally prefer to use regex in order to avoid problems with syntax changes in the future. That is, I think that checking that there is a line that contains `db` and the actual name of the Database would suffice:
```suggestion
      - {{ printf "/db.*%s/" .Env.JOOMLA_DATABASE_NAME }}
```
WDYT?